web/atlas.json

```diff
@@ -1059,7 +1059,6 @@
 {"id": "twqxc9", "name": "Tails Gets Trolled", "description": "The official logo and iconic Tails Gets Trolled face from the webcomic of the same name.", "website": "http://www.tailsgetstrolled.org", "subreddit": "/r/RealTailsGetsTrolled", "center": [1817.5, 1626.5], "path": [[1808.5, 1612.5], [1808.5, 1639.5], [1825.5, 1639.5], [1825.5, 1612.5]]},
 {"id": "twqxab", "submitted_by": "mamasbreads", "name": "Spanish coat of arms", "description": "Found on the Spanish flag, the coat of arms is composed of the four medieval kingdoms which formed Spain: Castilla (top-left), Leon (top-right), Aragon (bottom-left), and Navarra (bottom-right). The Spanish crown is on top of the coat of arms, while on either side are the Pillars of Hercules from Greek mythology.", "website": "https://en.wikipedia.org/wiki/Coat_of_arms_of_Spain", "subreddit": "/r/esPlace", "center": [1413.5, 296.5], "path": [[1396.5, 279.5], [1396.5, 313.5], [1430.5, 313.5], [1429.5, 279.5]]},
 {"id": "twqx4v", "submitted_by": "oski165", "name": "Helltaker", "description": "A picture of a character from a small Polish videogame Helltaker named Lucifer along with a small logo", "website": "", "subreddit": "", "center": [1990.5, 1254.5], "path": [[1981.5, 1237.5], [1981.5, 1271.5], [1998.5, 1271.5], [1999.5, 1237.5]]},
-{"id": "twqx0n", "submitted_by": "FakeGoonmachine", "name": "Ironman Mode", "description": "These three helmets represent 3 'Ironman' game modes within RuneScape: Standard, Hardcore, and Ultimate. Ironman Mode is an account type that requires the player to be entirely self-sufficient. Ironmen are locked out of or restricted in most forms of interactions with other players such as trading, the Grand Exchange, PvP, most group minigames, and almost all other group activities.", "website": "https://oldschool.runescape.wiki/w/Ironman_Mode", "subreddit": "/r/ironscape, /r/2007scape", "center": [90.5, 42.5], "path": [[63.5, 50.5], [63.5, 34.5], [117.5, 34.5], [117.5, 50.5]]},

```

Comment:
This is valid as an encompassing entry for all three Ironmade modes.

Reply:
Sounds fine, no need to implement this commit.